http-server/build.gradle

```diff
@@ -1,22 +1,43 @@
 plugins {
     id 'application'
     id 'com.github.johnrengelman.shadow' version '4.0.4'
-    id 'de.undercouch.download' version '3.4.3'
 }
 
 archivesBaseName = "$group-$name"
 description = 'TripleA DropWizard Http Server'
-mainClassName = '' // org.triplea.game.client.HeadedGameRunner'
-
+mainClassName = 'org.triplea.server.http.ServerApplication'
 ext {
     releasesDir = file("$buildDir/releases")
 }
 
+
+configurations {
+    testImplementation {
+        // database-rider brings in slf4j-simple as a transitive dependency
+        // DropWizard has logback baked in and cannot have multiple slf4j bindings.
+        exclude group: 'org.slf4j', module: 'slf4j-simple'
```

Comment:
It's always great having to deal with conflicting dependencies ^^

Reply:
Oh yeah, the no-op SLF4J logger was a nice icing as in some circumstances there was no warning about multiple SLF4J binding. #HeadSeeKeyboard :face_with_head_bandage: